src/helpers.js

```diff
@@ -62,19 +62,19 @@ export const getClipboardString = string => string
 const textColor = `#202020`;
 
 const defaultOptions = {
-    containerPadding: '0 1rem 1rem',
+    containerPadding: '0 1rem 0 0',
     containerMarginBottom: '2rem',
     displayCodeWidth: '80%',
     copyButtonWidth: '20%',
-    copyButtonHeight: '2rem',
+    copyButtonPadding: '1rem 0',
     copyButtonOutline: `2px solid`,
     copyButtonFontSize: '1rem'
 };
 
 const defaultColors = {
     background: '#f7f7f7',
     borderColor: 'rgba(0, 0, 0, 0.1)',
-    textColor: textColor,
+    textColor
```

Comment:
This will only work on newer browsers; if we want to support older browsers we would need to add babel or typescript to transpile.  Come to think of it, `const`, `let`, and template strings are the same deal.